docs/error-messages/compiler-errors-1/compiler-error-c2162.md

```diff
@@ -20,7 +20,4 @@ The following example generates C2162:
 // compile with: /c
 #define make_string1(s) #    // C2162
 #define make_string2(s) #s   // OK
-
-#define make_char1(c) #@    // C2162
-#define make_char2(c) #@c   // OK
```

Comment:
This is the new part added which shows the expected macro formal parameter error similar to the `make_string[1,2]` macros, just with the charizing operator (`#@`) instead. But I do see how it might be a little misleading, so we can leave it out and maintain the status quo for now.